src/System.Net.Security/tests/FunctionalTests/NegotiateStreamTestForUnix.cs

```diff
@@ -73,31 +73,14 @@ public bool CheckAndClearCredentials(ITestOutputHelper output)
             }
 
             // Clear the credentials
-            try
-            {
-                var startInfo = new ProcessStartInfo(KDestroyCmd);
-                startInfo.UseShellExecute = true;
-                startInfo.CreateNoWindow = true;
-                startInfo.Arguments = "-A";
-                using (Process clearCreds = Process.Start(startInfo))
-                {
-                    clearCreds.WaitForExit();
-                    output.WriteLine("kdestroy returned {0}", clearCreds.ExitCode);
-                    return (clearCreds.ExitCode == 0);
-                }
-            }
-            catch (Win32Exception)
-            {
-                // https://github.com/dotnet/corefx/issues/24000
-                // on these distros right now
-                Assert.True(PlatformDetection.IsUbuntu1704 ||
-                            PlatformDetection.IsUbuntu1710 ||
-                            PlatformDetection.IsOpenSUSE   ||
-                            PlatformDetection.IsFedora     ||
-                            PlatformDetection.IsDebian     ||
-                            PlatformDetection.IsCentos7);
```

Comment:
This is no longer necessary?

Reply:
I'm not sure @stephentoub I was hoping that dotnet-bot would tell me. Since all this was introduced at the same time, I'm assuming that root cause is same e.g. we broke process.Start with shell flag.